.gitignore

```diff
@@ -0,0 +1,5 @@
+.*.swo
+.*.swp
+
+server_standalone/server_standalone
```

Comment:
It's your call, doesn't really matter much each way, i'm just finding
things to nit pick while you land your change.

On Wed, Sep 9, 2015 at 3:06 PM, Mark Sheahan <notifications@github.com>
wrote:

> In .gitignore <https://github.com/pkg/sftp/pull/46#discussion_r39007718>:
>
> > @@ -0,0 +1,5 @@
> > +.*.swo
> > +.*.swp
> > +
> > +server_standalone/server_standalone
>
> Yes, I did my integration tests that way. I probably could have used go
> install but it's just habit...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pkg/sftp/pull/46/files#r39007718>.
>